libtcmu_log.c

```diff
@@ -226,11 +241,17 @@ void tcmu_dbg_message(const char *funcname, int linenr, const char *fmt, ...)
 
 static void log_output(int pri, const char *msg)
 {
-	log_to_syslog(pri, msg);
-	/* log to stdout if tcmu_log_level is DEBUG */
-	if (pri <= TCMU_LOG_DEBUG && tcmu_log_level == TCMU_LOG_DEBUG) {
-		fprintf(stdout, "%s", msg);
+	struct log_output *output;
+	char timestamp[TCMU_TIME_STRING_BUFLEN] = {0, };
```

Comment:
Between the initial variable declarations and other code, could add a space so the coding style is like the other code.